utils/training.py

```diff
@@ -53,6 +54,34 @@ def lr_schedule(epoch):
 
   return lr_schedule
 
+def visualize_mixture_weights(model, domains, num_templates, num_layers):
+  """ Returns a matrix of mixture weights for visualization.
+
+  Arguments:
+  model: target model.
+  domains: list of domain names.
+  num_templates: number of templates in the model.
+  num_layers: number of resblock layers in the model.
+  """
+  num_domains = len(domains)
+  mix_w_matr = np.zeros((num_layers, num_domains * (num_templates + 1)))
```

Comment:
Maybe slightly expand / clarify this variable name? Later you use a mixture of `mw` and `mix_w`, could pick one (super optional).